inf-clojure.el

```diff
@@ -773,7 +787,7 @@ prefix argument PROMPT-FOR-NS, it prompts for a namespace name."
   (let ((ns (if prompt-for-ns
                 (car (inf-clojure-symprompt "Ns vars" (clojure-find-ns)))
               (clojure-find-ns))))
-    (comint-proc-query (inf-clojure-proc) (format inf-clojure-ns-vars-form ns))))
+    (inf-clojure--send-string (inf-clojure-proc) (format (inf-clojure-ns-vars-form) ns))))
```

Comment:
This is a very good point and the subject of #55 , who knows what is the user going to do? Now we set the repl type in the right places, even in more than necessary maybe. Better be safe 😀

However it looks like there is a reason why it is like this: https://github.com/emacs-mirror/emacs/blob/master/lisp/comint.el#L2899: to receive something back from the function instead of showing it in the buffer.

I have also discovered that when you set a process filter (which we do there) the output is not pasted to the buffer.

So please let me to some further trial on this one.